project/REVIEWING.md

```diff
@@ -207,3 +197,22 @@ review session. The goal of that session is to agree on one of the following out
  * Escalate to Solomon by formulating a few specific questions on which his answers will allow
    maintainers to decide.
 
+## Milestones
+
+Typically, every merged pull request get shipped naturally with the next release cut from the
+`master` branch (either the next minor or major version). However, the time-based nature of the
+release process provides no guarantee that a given pull request will get merged in time. In other
+words, all open pull requests are implicitly considered part of the next minor or major release
+milestone, and this won't be materialized on GitHub.
+
+A merged pull request must be attached to the milestone corresponding to the release in which it
```

Comment:
Yep, I think we're going need to automate that (we had a brief discussion with @thaJeztah about it this morning).